docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -0,0 +1,157 @@
+# Integrate with SAP Cloud Logging
+
+The SAP Cloud Logging service is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces. This guide explains how to define LogPipelines and TracePipelines in order to ingest application and access logs as well as distributed trace data in the SAP Cloud Logging service.
+
+## Prerequisites
+
+- A SAP Cloud Logging service with OpenTelemetry enabled to ingest distributed traces
+- A secret named `cls` in `cls-integration` namespace, holding the credentials and endpoints for the instance
+
+## Ship Logs to Cloud Logging Service
+
+The telemetry module supports convenient shipment of application and access logs using LogPipeline custom resources (see [Kyma Telemetry Application Logs Documentation](./../../02-logs)). The setup distinguishes application logs and access logs which can be configured independently.
+To enable shipping logs to Cloud Logging Service follow the below procedure:
```

Comment:
```suggestion
To enable shipping logs to the SAP Cloud Logging service follow the below procedure:
```
I would stick to using the same form for the service throughout the entire document. I guess that would be either "the SAP Cloud Logging service" or "SAP Cloud Logging".